CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 # For MSVC_RUNTIME_LIBRARY
 cmake_minimum_required(VERSION 3.15)
 
+if (MSVC)
```

Comment:
Hi, you need to define what is `MSVC` here

Reply:
It should be predefined by `cmake`. See: https://cmake.org/cmake/help/latest/variable/MSVC.html

Reply:
Wait. I haven't found [this](https://stackoverflow.com/questions/41692725/cmake-doesnt-recognize-msvc-compiler) last time.